features/kafka/producer/src/main/java/org/opennms/features/kafka/producer/OpennmsKafkaProducer.java

```diff
@@ -96,6 +100,11 @@
     private final AlarmEqualityChecker alarmEqualityChecker =
             AlarmEqualityChecker.with(AlarmEqualityChecker.Exclusions::defaultExclusions);
 
+    // TODO: Determine an appropriate queue capacity
+    private final BlockingQueue<KafkaRecord> kafkaSendQueue =
```

Comment:
Can we expose the queue size as a property in the blueprint so that it can be configured?